index.js

```diff
@@ -16,3 +16,59 @@ async function start() {
   // Now try and complete the program.
   process.exit();
 }
+fconst readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+start();
```

Comment:
you're also calling your start() function twice so it's asking for my secret number twice, you can get rid of the top one.